pkg/resourcetopologyexporter/resourcetopologyexporter.go

```diff
@@ -44,19 +47,34 @@ func Execute(cli podresourcesapi.PodResourcesListerClient, nrtupdaterArgs nrtupd
 		return err
 	}
 
-	resMon, err := NewResourceMonitor(cli, resourcemonitorArgs, rteArgs)
+	cs, err := nrtupdater.GetK8sClient("")
+	if err != nil {
+		return err
+	}
+
+	var condChan chan v1.PodCondition
+	if rteArgs.PodReadinessEnable {
+		condChan = make(chan v1.PodCondition)
+		condIn, err := podreadiness.NewConditionInjector(cs)
```

Comment:
the `ConditionInjector` is the sole owner of the `clientset` so it should probably create itself

Reply:
Ack